File: src/main/java/com/kata/board/dto/PostDto.java
package com.kata.board.dto;

import jakarta.persistence.Column;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import java.time.LocalDateTime;

public class postList {
    private Long id;
    private String title;
    private String userName;
    private int view;
    private LocalDateTime createdDate;
    private LocalDateTime modifiedDate;
}

File: src/main/kotlin/com/kata/board/dto/PostDto.kt
package com.kata.board.dto

import com.kata.board.entity.Post
import jakarta.persistence.Entity
import java.time.LocalDateTime


data class PostDto (
    val id: Long? = null,
    val title: String = "",
    val createdDate: LocalDateTime? = null,
    val modifiedDate: LocalDateTime? = null,
    var content: String = "",
    var view: Int? = null,
) {
    fun convertPostDtoFromPost(): Post {
        return Post(
            id = this.id,
            title = this.title,
            content = this.content,
            )
    }
}